inst/WORDLIST

```diff
@@ -1,5 +1,6 @@
 analytics
 aut
+bcausse
```

Comment:
wat?

Reply:
Yeah I should have a look at the WORDLIST if I want to use it.